src/lib/arrays.js

```diff
@@ -0,0 +1,67 @@
+exports.getNthElement = (index, array) => {
+  return array[index % array.length]; // remainder of index/array length equals given parameter in questions
+};
+
+exports.arrayToCSVString = array => {
+  return array.toString();
+};
+
+exports.csvStringToArray = string => {
+  return string.split(',');
+};
+
+exports.addToArray = (element, array) => {
+  array.push(element);
+};
+
+exports.addToArray2 = (element, array) => {
+  let expectedArray = array.concat(element);
+  return expectedArray;
+};
+
+exports.removeNthElement = (index, array) => {
+  return array.splice(2, 1);
+};
+
+exports.numbersToStrings = numbers => {
+  return numbers.map(String);
+};
+
+exports.uppercaseWordsInArray = strings => {
+  return strings.map(expected => expected.toUpperCase());
```

Comment:
when you map over an array, best practice to put the singular of the word of the array, so in this case it would be string, not expected. Still technically works no matter what you put though :)